### DISCUSSION TITLE
Placeholder/default for Custom CSS Setting

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#4365] [Archive]
Is it possible to have a placeholder or default value that gives an example of how to customize a value such as `--primary`? If not, we could add an interface option to code for "snippets" that could be displayed as text links below the interface for injecting preset code snippets. Then we could use that.

**Comment by philleepflorence** [comment]
Is this more related to [Styles Guide: Custom CSS](https://docs.directus.io/guides/styles/#custom-css) or more as a helper to update the custom CSS for individual [Interfaces](https://docs.directus.io/concepts/interfaces/)?

**Reply by benhaynes** [reply]
A helper, so it's easier to add custom values without requiring the Docs.
